libmisc/csrand.c

```diff
@@ -59,3 +65,23 @@ csrand(void)
 	fprintf(log_get_logfd(), _("Unable to obtain random bytes.\n"));
 	exit(1);
 }
+
+
+/*
+ * Return a uniformly-distributed CS random value in the interval [0, n-1].
+ */
+unsigned long
+csrand_uniform(unsigned long n)
+{
+	unsigned long  r, max, mask;
```

Comment:
Can you rename `r` to something more meaningful? `random` perhaps?

Reply:
the thing is `rand` and `random` are both reserved names (from `<stdlib.h>`).  Maybe I could better clarify it in a different way if it's not very clear?

Reply:
ok, no problem